Classes/Misc_Player.uc

```diff
@@ -19,12 +19,7 @@ var config bool bDisableAmmoRegen;
 var config bool bShowTeamInfo;          // show teams info on the HUD
 var config bool bExtendedInfo;          // show extra teammate info
 
-var config enum EDamageIndicator

```

Comment:
I'd prefer to keep the enum but didn't feel like writing a complex function to convert ints to enums because I've been babied by C# doing this for me the past 7 years.